### PR TITLE
(tlg0007.tlg077.perseus-eng2.xml) EpiDoc and CTS Conversion

### LINKED ISSUE #640: (tlg0007.tlg077.perseus-eng2.xml) EpiDoc and CTS Conversion
Claiming file for conversion work.

### PR DESCRIPTION
**Pull request by kcooper42**
Fix #640 
Edited file for EpiDoc and CTS compliance.

### CODE REVIEW

File: data/tlg0007/tlg077/tlg0007.tlg077.perseus-eng4.xml [hunk 1]
```diff
@@ -0,0 +1,1140 @@
...
+   </teiHeader>
+   <text xml:lang="eng">
+      <body>
+      	<div type="translation" xml:lang="eng" n="urn:cts:greekLit:tlg0007.tlg077.perseus-eng4">
+         <pb xml:id="v.1.p.251"/>
+      		<head>Plutarch's Rules for the Preservation of Health.</head><lb/>
+      		<head>A Dialogue.</head><lb/>
+      		<head>MOSCHIO, ZEUXIPPUS.</head>
+         <div type="textpart" subtype="section" n="1">
+         	<p rend="indent"><said who="#Moschion"><label>MOSCHION.</label> And you, Zeuxippus, diverted Glaucus the
```
Comment (lcerrato) [inline data/tlg0007/tlg077/tlg0007.tlg077.perseus-eng4.xml]: I think that this translation calls him Moschio? So the tag and label should reflect that.

File: data/tlg0007/tlg077/tlg0007.tlg077.perseus-eng4.xml [hunk 2]
```diff
@@ -0,0 +1,1140 @@
...
+				<state unit="stephpage" n="chunk"/>
+				</refsDecl>--></encodingDesc>
+      <profileDesc>
+         <langUsage>
+            <language ident="eng">English</language>
+            <language ident="grc">Greek</language>
+         	<language ident="lat">Latin</language>
+         </langUsage>
+      	<particDesc>
+      		<person><persName>Moschion</persName></person>
```
Comment (lcerrato) [inline data/tlg0007/tlg077/tlg0007.tlg077.perseus-eng4.xml]: See my second comment on the name rendering here.

**Comment by lcerrato** [comment]
@kcooper42 This is really great!

**Comment by kcooper42** [comment]
@lcerrato I've corrected the names to Moschio

**Pull request merged**